tests/manifold/transfinite_manifold_03.output

```diff
@@ -2,20 +2,20 @@
 DEAL::Testing with PolarManifold dim=2, spacedim=2
 DEAL::80   cells, degree=1: 3.06146745892 
 DEAL::80   cells, degree=2: 3.14143771670 
-DEAL::80   cells, degree=3: 3.14159250174 
-DEAL::80   cells, degree=4: 3.14159265350 
+DEAL::80   cells, degree=3: 3.14159250173 
+DEAL::80   cells, degree=4: 3.14159265349 
 DEAL::
 DEAL::Testing with SphericalManifold dim=2, spacedim=2
 DEAL::80   cells, degree=1: 3.06146745892 
-DEAL::80   cells, degree=2: 3.14064643171 
-DEAL::80   cells, degree=3: 3.14158597871 
-DEAL::80   cells, degree=4: 3.14159257389 
+DEAL::80   cells, degree=2: 3.14143771670 
+DEAL::80   cells, degree=3: 3.14159250173 
+DEAL::80   cells, degree=4: 3.14159265350 
 DEAL::
 DEAL::Testing with SphericalManifold dim=3, spacedim=3
 DEAL::56   cells, degree=1: 3.21034343105 
-DEAL::56   cells, degree=2: 4.14326491350 
-DEAL::56   cells, degree=3: 4.18822553211 
-DEAL::56   cells, degree=4: 4.18868242879 
+DEAL::56   cells, degree=2: 4.18001126295 
+DEAL::56   cells, degree=3: 4.18863487067 
+DEAL::56   cells, degree=4: 4.18878933389 
```

Comment:
That's cool to see how much better we are now with the new manifold. That definitely looks convincing!